sh_scrapy/settings.py

```diff
@@ -76,6 +76,28 @@ def _get_action_on_missing_addons(settings):
     return on_missing_addons
 
 
+def _update_old_classpaths(settings):
+    """Update user's project settings with proper class paths"""
+    for setting_key in settings.attributes.keys():
+        setting_value = settings[setting_key]
+        if hasattr(setting_value, 'copy_to_dict'):
```

Comment:
A workaround to make it work for both Scrapy 1.0.5 (with dicts as values) and Scrapy 1.1.1 (with `BaseSettings` as values).